README.textile

```diff
@@ -28,4 +27,4 @@ Hidoyatov Victor, Troshin Sergey, Svinukhov Vladimir, "Ilia Dubinin":https://git
 "Aleksey Grigirov":https://github.com/KTannenberg, Alexander Berezovsky, "Sergey Burtsev":https://github.com/burtsevsergey, "Baratali Izmailov":https://github.com/baratali, "Max Vetrenko":https://github.com/maxvetrenko,
 "Pavel Baranchikov":https://github.com/pbaranchikov , "Ashutosh Agarwal":https://github.com/Radsaggi, "Alexey Nesterenko":https://github.com/alexkravin. , ......  for whole list please look at "contributors list":https://github.com/sevntu-checkstyle/sevntu.checkstyle/network/members .
 
-The "Jsr305AnnotationsCheck" is based on the "JSR305CheckstylePlugin":https://github.com/bjrke/JSR305CheckstylePlugin by "Jan Burkhardt":https://github.com/bjrke with contributions from Mattias Nissler, Thorsten Ehlers, Fabian Loewner and Ole Langbehn.
\ No newline at end of file
+The "Jsr305AnnotationsCheck" is based on the "JSR305CheckstylePlugin":https://github.com/bjrke/JSR305CheckstylePlugin by "Jan Burkhardt":https://github.com/bjrke with contributions from Mattias Nissler, Thorsten Ehlers, Fabian Loewner and Ole Langbehn.
```

Comment:
I am not seeing what the change is here.

Reply:
There were no new line symbol at the end of file:
```
git diff README.textile
diff --git a/README.textile b/README.textile
index 6efab66b..aabae016 100644
--- a/README.textile
+++ b/README.textile
@@ -2,7 +2,6 @@ h3. Information "!https://secure.travis-ci.org/sevntu-checkstyle/sevntu.checksty
 
 sevntu-checks: !https://img.shields.io/maven-central/v/com.github.sevntu-checkstyle/sevntu-checks.svg?label=Maven%20Central!:https://search.maven.org/search?q=g:%22com.github.sevntu-checkstyle%22%20AND%20a:%22sevntu-checks%22
 sevntu-checkstyle-idea-extension:!https://img.shields.io/maven-central/v/com.github.sevntu-checkstyle/sevntu-checkstyle-idea-extension.svg?label=Maven%20Central!:https://search.maven.org/search?q=g:%22com.github.sevntu-checkstyle%22%20AND%20a:%22sevntu-checkstyle-idea-extension%22
-sevntu-checkstyle-maven-plugin:!https://img.shields.io/maven-central/v/com.github.sevntu-checkstyle/sevntu-checkstyle-maven-plugin.svg?label=Maven%20Central!:https://search.maven.org/search?q=g:%22com.github.sevntu-checkstyle%22%20AND%20a:%22sevntu-checkstyle-maven-plugin%22
 sevntu-checkstyle-sonar-plugin:!https://img.shields.io/maven-central/v/com.github.sevntu-checkstyle/sevntu-checkstyle-sonar-plugin.svg?label=Maven%20Central!:https://search.maven.org/search?q=g:%22com.github.sevntu-checkstyle%22%20AND%20a:%22sevntu-checkstyle-sonar-plugin%22
 eclipsecs-sevntu-plugin: "all versions":https://github.com/sevntu-checkstyle/sevntu.checkstyle/tree/gh-pages/update-site/plugins
 
@@ -28,4 +27,4 @@ Hidoyatov Victor, Troshin Sergey, Svinukhov Vladimir, "Ilia Dubinin":https://git
 "Aleksey Grigirov":https://github.com/KTannenberg, Alexander Berezovsky, "Sergey Burtsev":https://github.com/burtsevsergey, "Baratali Izmailov":https://github.com/baratali, "Max Vetrenko":https://github.com/maxvetrenko,
 "Pavel Baranchikov":https://github.com/pbaranchikov , "Ashutosh Agarwal":https://github.com/Radsaggi, "Alexey Nesterenko":https://github.com/alexkravin. , ......  for whole list please look at "contributors list":https://github.com/sevntu-checkstyle/sevntu.checkstyle/network/members .
 
-The "Jsr305AnnotationsCheck" is based on the "JSR305CheckstylePlugin":https://github.com/bjrke/JSR305CheckstylePlugin by "Jan Burkhardt":https://github.com/bjrke with contributions from Mattias Nissler, Thorsten Ehlers, Fabian Loewner and Ole Langbehn.
\ No newline at end of file
+The "Jsr305AnnotationsCheck" is based on the "JSR305CheckstylePlugin":https://github.com/bjrke/JSR305CheckstylePlugin by "Jan Burkhardt":https://github.com/bjrke with contributions from Mattias Nissler, Thorsten Ehlers, Fabian Loewner and Ole Langbehn.
```